common/metrics/metric_defs.go

```diff
@@ -911,6 +911,10 @@ var (
 	ShardLingerSuccess                             = NewTimerDef("shard_linger_success")
 	ShardLingerTimeouts                            = NewCounterDef("shard_linger_timeouts")
 	DynamicRateLimiterMultiplier                   = NewGaugeDef("dynamic_rate_limit_multiplier")
+	DLQWrites                                      = NewCounterDef(
+		"dlq_writes",
+		WithDescription("The number of times a history task is enqueued to a DLQ"),
```

Comment:
This will be used by the replication queue as well, so it's actually the number of times anything is written to the DLQ. I'd also include a mention of `tdbg dlq`, so admins know where to look further.